src/Raven.Server/Commercial/LicenseManager.cs

```diff
@@ -602,7 +602,7 @@ private async Task<License> GetUpdatedLicenseForActivation(License currentLicens
                     if (_skipLeasingErrorsLogging == false && Logger.IsInfoEnabled)
                     {
                         // ReSharper disable once MethodHasAsyncOverload
-                        Logger.Info("Skipping updating of the license from string or path or from api.ravendb.net because 'Licensing.DisableAutoLicenceUpdate' was set to true");
+                        Logger.Info("Skipping updating of the license from string or path or from api.ravendb.net because 'Licensing.DisableAutoUpdate' was set to true");
```

Comment:
You can do it even slightly better and use `RavenConfiguration.GetKey(x => x.Licensing.DisableAutoUpdate)` instead of hard coded string